tests/tools/fio/workload.go

```diff
@@ -13,6 +13,9 @@ import (
 // WriteFiles writes files to the directory specified by path, up to the
 // provided size and number of files.
 func (fr *Runner) WriteFiles(relPath string, opt Options) error {
+	fr.PathLock.Lock(relPath)
+	defer fr.PathLock.Unlock(relPath)
```

Comment:
I guess these will have to change to something like
```
   defer fr.PathLoc.Lock(relpath).Unlock()
```

Reply:
yep I'll update the invocations here once the upstream PR merges.